Utilities/python/requirements.txt

```diff
@@ -0,0 +1,3 @@
+black==24.10.0
```

Comment:
thought: this file is not currently in use, however we should have a way to define various python dependencies we require.

The build-using-self does not currently have any dependencies, however we should determine a way to run black formatting, flake8 and mypy checks on the various python files/modules in the repository.

Reply:
Removed the requirements.txt file